packages/apps/spaces/app/organizations/src/components/Columns/Cells/index.ts

```diff
@@ -0,0 +1,8 @@
+export * from './avatar';
```

Comment:
was the nextjs problem with imports and tree shaking resolved?

Reply:
If it was not I would opt for avoiding index files that export multiple entities unless they are used in single place

Reply:
They are all used in the single same place at the same time. There's nothing to treeshake from here. I was having literally 1 import for every thing and half of the file was only imports.